draft-ietf-quic-tls.md

```diff
@@ -1133,27 +1133,6 @@ sets available to a server one-round trip earlier than parameters that are
 carried in QUIC packets.  This document does not define that extension.
 
 
-## Source Address Validation {#source-address}
```

Comment:
With some massaging, I think this text should be moved to the bottom of the Proof of Source Address Ownership section in the transport doc.